### PR TITLE
fix: asset transaction history

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

tl;dr /asset/<assetId>transactions (both for regular and NFT assets) got rugged since:

https://github.com/shapeshift/web/blob/0f969a3376d505347c8428c5485aa5057d6836ed/src/hooks/useRouteAssetId/useRouteAssetId.ts#L11

The reason for that is the path above is conflicting with `/:chainId/:assetSubId/transactions` subroute:

https://github.com/shapeshift/web/blob/999c31553b3bf684bc0487b86d0ff20bc5dc437a/src/Routes/RoutesCommon.tsx#L64

We cannot have transactions as a nested route anymore and need to lift it up as top-level routes for both the assetId and nftId flavors 

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

None, but see https://discord.com/channels/554694662431178782/1131323172369743884/1131336866315108554

## Risk

/asset routing may be borked

## Testing

- asset pages are happy
- account asset pages are happy
- asset transactions pages are happy
- account asset transactions pages are happy

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

## Screenshots (if applicable)

- Prod

Screenshot 2023-07-20 at 00 14 23

Screenshot 2023-07-20 at 00 14 07

(Note how appending /transactions to the asset URL does nothing)

Screenshot 2023-07-20 at 00 13 29
Screenshot 2023-07-20 at 00 12 41

- This diff
Screenshot 2023-07-20 at 00 17 06
Screenshot 2023-07-20 at 00 16 32
Screenshot 2023-07-20 at 00 15 59
Screenshot 2023-07-20 at 00 14 58

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4955** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4955" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4955?utm_source=stack-comment).

**Review by 0xdef1cafe — APPROVED** [review]

**Comment by 0xdef1cafe** [comment]
looks sane will let ops test
